qa/L0_DALI_GPU_ensemble/model_repository/dali_2/pipeline.py

```diff
@@ -0,0 +1,43 @@
+# The MIT License (MIT)
```

Comment:
Can you unify with `qa/L0_DALI_GPU_ensemble/model_repository/dali_1/pipeline.py`. You can add a switch selecting which pipeline is used.

Reply:
You mean merging it into one file? What is a benefit of doing this? 
I put those file in the model directories thinking that in the future that might simplify automating testing against DALI pipelines (e.g. automatically importing pipelines for each model and running them)

Reply:
To reduce the repetition, but just a suggestion.